TypeCobol.LanguageServer/Workspace.cs

```diff
@@ -644,6 +644,7 @@ public void UpdateMissingCopies(Uri fileUri, List<string> RemainingMissingCopies
         public void RefreshOpenedFiles()
         {
             RefreshCustomSymbols();
+            this.CompilationProject?.ClearImportedCompilationDocumentsCache();
```

Comment:
`CompilationProject` can't be `null`. It is safe to call the method without check.

Reply:
ok done in 0fd8647